binding/java/src/ai/picovoice/rhino/Utils.java

```diff
@@ -118,7 +118,7 @@ private static Path extractResources(Path jarPath) throws IOException {
         return resourceDirectoryPath;
     }
 
-    private static String getEnvironmentName() throws RuntimeException {
+    public static String getEnvironmentName() throws RuntimeException {
```

Comment:
I believe if you remove public and just have it as `static String..` then it's package-private, which would mean the user can't use it, but we can.